src/test/java/guitests/MenuControlTest.java

```diff
@@ -117,6 +117,9 @@ public void menuControlTest() {
 
         // Testing First Panel selected
         assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        press(KeyCode.CONTROL).press(KeyCode.DIGIT1).release(KeyCode.DIGIT1).release(KeyCode.CONTROL);
+        assertEquals(0, (int) panelControl.getCurrentlySelectedPanel().get());
+        // TODO (currently not possible) test that panelsScrollPane.getHValue() is 0
```

Comment:
I know, but I think it is better in the long run that we have a reference to the ScrollPane in UI/PanelControl, and eventually remove the one in MenuControl. The former is really just an implementation detail of the panels, and it isn't conceptually part of the menu.

The name seems weird to me because MenuControl is just a subclass of JavaFX's MenuBar... it can't really have any meaningful tests without going into the functionality of its items.